src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span class="image">
+        <img
+          class="image_logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        >
+      </span>
```

Comment:
better use `a` tag instead os span.
Image should be a part of header block.
Add changes to styles according to changes in html.
```suggestion
      <a class="header__image">
        <img
          class="header__image-logo"
          src="./images/logo.png"
          alt="Moyo logo"
        >
      </a>
```